src/main/webapp/script.js

```diff
@@ -260,3 +280,137 @@ function checkPercentage(headerColumn) {
   }
   return false;
 }
+
+// Returns an object containing all of the relevant data in order
+// to render the geoJson map of the counties.
+function getMapsData(censusDataArray) {
+  const countyToPopMap = new Map();
+  const populationsList = [];
+  // Get rid of the header
+  const censusArray = censusDataArray.slice(1);
+  Array.prototype.forEach.call(censusArray, (county) => {
```

Comment:
I think this can be shortened to censusArray.forEach((county) => { ... }) like done here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach

Reply:
For some reason censusArray.forEach() was throwing an error earlier but I changed it back and now it works.